src/govuk/components/back-link/_back-link.scss

```diff
@@ -4,6 +4,15 @@
 
 @include govuk-exports("govuk/component/back-link") {
 
+  // Size of chevron (excluding border)
+  $chevron-size: 7px;
+
+  // Size of chevron border
+  $chevron-border-width: 1px;
+
+  // Colour of chevron
+  $chevron-border-colour: $govuk-text-colour;
```

Comment:
The chevron border colour here is different to that in the breadcrumbs. If we want to be consistent with the breadcrumbs this should be `$govuk-secondary-text-colour`

Reply:
This sort of makes sense to me – in the breadcrumbs, the arrow is a separator and isn't part of the link.

![image](https://user-images.githubusercontent.com/121939/79850323-a5ae2d80-83bb-11ea-8a44-9f17a6edd009.png)

With the back link, it is treated as part of the link.

![image](https://user-images.githubusercontent.com/121939/79850301-9af39880-83bb-11ea-9ead-f1667142963c.png)

So in that sense, the arrow in the back link is interactive, and in the breadcrumbs it's not.

Thoughts?

Reply:
Fair point yeah. I suppose I'd ask why the arrow should be interactive in the back link?

Reply:
Hmm, this is a subtle one. I think on balance the chevrons should both be the same colour, for visual consistency. What they signify in each case is very slightly different, and for that reason I think it's OK to keep the active areas as they currently are.

Reply:
Thanks both! I've changed the colour for the chevron to be `$govuk-secondary-text-colour` by default, and `$govuk-text-colour` on focus.